index.html

```diff
@@ -237,6 +237,36 @@ <h3>
               </p>
             </dd>
           </dl>
+          <p>
+            The Working Group may begin work on the following W3C normative specifications:
+          </p>
+
+          <dl>
+            <dt >Decentralized Identifier (DID) Method Specification</dt>
+           <dd>
+             <p>
+             DID Method Specifications may be produced by the working group.
+             </p>
+             <p>
+             The <a href="https://www.w3.org/TR/did-spec-registries/#did-methods">DID Specification Registries</a> contains many links to examples of DID Method Specifications, some of which may be selected by the working group for development into a Recommendation. The group might also choose a similar starting document not listed in the Registry.
+             </p>
+             <p>
+             It is not intended that any DID Method Specification adopted by the working group will advance beyond <a href="https://www.w3.org/2021/Process-20211102/#RecsWD">Working Draft status</a>.
```

Comment:
Suggest "developed" instead of "adopted".

Reply:
Your recommended changes have been made. Could you re-review?